gui/qt/qrreaderutil.py

```diff
@@ -0,0 +1,14 @@
+from PyQt5.QtWidgets import QMessageBox
+
+# TODO: Remove this when it is unlikely to encounter Qt5 installations that are
+# missing QtMultimedia
+def warn_unless_can_import_qrreader():
```

Comment:
Add a parameter, `window` which should be documented to be a `MessageBoxMixin` instance.

Reply:
**Update**: Move this into `gui/qt/__init__.py`, alongside the other "warning" style functions, doing similar to what they do.

It can optionally take a second parameter `show_warning=True`, if False, won't show a warning but tell you if it *would* have shown it.  You can use that to gray out the QComboBox in the prefs... without actually displaying a warning.